### PR TITLE
Document Update

### PR DESCRIPTION
**Pull request by NiteshKoushik**
Therfore to Therefore

[ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/CAMEL) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
[ ] Each commit in the pull request should have a meaningful subject line and body.
[ ] If you're unsure, you can format the pull request title like `[CAMEL-XXX] Fixes bug in camel-file component`, where you replace `CAMEL-XXX` with the appropriate JIRA issue.
[ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
[ ] Run `mvn clean install -Psourcecheck` in your module with source check enabled to make sure basic checks pass and there are no checkstyle violations. A more thorough check will be performed on your pull request automatically.
Below are the contribution guidelines:
https://github.com/apache/camel/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: docs/components/modules/ROOT/pages/properties-component.adoc [hunk 1]
```diff
@@ -31,7 +31,7 @@ The component supports 10 options, which are listed below.
 | *camel.component.properties.ignore-missing-location* | Whether to silently ignore if a location cannot be located, such as a properties file not found. | false | Boolean
 | *camel.component.properties.initial-properties* | Sets initial properties which will be used before any locations are resolved. The option is a java.util.Properties type. |  | String
 | *camel.component.properties.location* | A list of locations to load properties. You can use comma to separate multiple locations. This option will override any default locations and only use the locations from this option. |  | String
-| *camel.component.properties.override-properties* | Sets a special list of override properties that take precedence and will use first, if a property exist. The option is a java.util.Properties type. |  | String
+| *camel.component.properties.override-properties* | Sets a special list of override properties that take precedence and will use first, if a property exists. The option is a java.util.Properties type. |  | String
```
Comment (oscerd) [inline docs/components/modules/ROOT/pages/properties-component.adoc:34]: This needs to be done in the javadoc of the componet

**Review by oscerd — CHANGES_REQUESTED** [review]
I reported just one update that needs to be done in the javadoc

File: docs/components/modules/ROOT/pages/properties-component.adoc [hunk 2]
```diff
@@ -368,7 +368,7 @@ The default mode is that both of them are in override mode, and they are check i
 The check stops at first found property value for the key.
 
 You can control these modes using the `systemPropertiesMode` and `environmentVariableMode`
-options on the properties component.
+options on the property's component.
```
Comment (jamesnetherton) [inline docs/components/modules/ROOT/pages/properties-component.adoc:371]: IMO the original wording is correct given that the name of the component is `properties`.

Comment (NiteshKoushik) [reply docs/components/modules/ROOT/pages/properties-component.adoc:371]: sure thanks.

**Comment by NiteshKoushik** [comment]
Hello @oscerd , is the below link is Javadoc file for this page, when I checked, I got this page - but feel the changes are not present in it, if this is the actual javadoc file.
https://github.com/apache/camel/blob/master/core/camel-api/src/main/java/org/apache/camel/spi/PropertiesComponent.java

**Comment by oscerd** [comment]
https://github.com/apache/camel/blob/master/core/camel-base/src/main/java/org/apache/camel/component/properties/PropertiesComponent.java

It should be this one.

**Comment by NiteshKoushik** [comment]
These changes are not present in javadoc file. I have reverted the changes as per the comments and also I have removed the changes which are done in the Javadoc file.
https://github.com/apache/camel/blob/master/core/camel-base/src/main/java/org/apache/camel/component/properties/PropertiesComponent.java

**Review by oscerd — APPROVED** [review]

**Pull request merged**